tests/test_reckless.py

```diff
@@ -159,6 +198,26 @@ def test_install(node_factory):
     assert os.path.exists(plugin_path)
 
 
+@unittest.skipIf(VALGRIND, "virtual environment triggers memleak detection")
+def test_poetry_install(node_factory):
+    """test search, git clone, and installation to folder."""
+    n = get_reckless_node(node_factory)
+    r = reckless([f"--network={NETWORK}", "-v", "install", "testplugpyproj"], dir=n.lightning_dir)
+    assert r.returncode == 0
+    assert 'dependencies installed successfully' in r.stdout
+    assert 'plugin installed:' in r.stdout
+    assert 'testplugpyproj enabled' in r.stdout
+    check_stderr(r.stderr)
+    plugin_path = Path(n.lightning_dir) / 'reckless/testplugpyproj'
+    print(plugin_path)
+    assert os.path.exists(plugin_path)
+    n.start()
+    print(n.rpc.testmethod())
+    assert n.daemon.is_in_log(r'plugin-manager: started\([0-9].*\) /tmp/ltests-[a-z0-9_].*/test_poetry_install_1/lightning-1/reckless/testplugpyproj/testplugpyproj.py')
+    assert n.rpc.testmethod() == 'I live.'
+
+
+@unittest.skipIf(VALGRIND, "virtual environment triggers memleak detection")
```

Comment:
Are all these skipped if VALGRIND is set? If yes there is a way to disable tests in the entire file with the following anywhere before the first test:

```python
if VALGRIND:
    pytest.skip(allow_module_level=True)
```